test/basic-images.test

```diff
@@ -564,6 +564,14 @@ test_expect_success !includepath "includepath5" "
 "
 
 
+exec_test_set_prereq fiptool
```

Comment:
Right, that's not yet available in the automated tests. I'll run it locally for now and add an ubuntu-22.04 check when it's available.

Reply:
Ok.